### ISSUE TITLE
Outstanding WOs -- Electrical Model

### ISSUE THREAD

**Issue by tnjtr2** [tnjtr2/TrafficEngineering#49]

**Comment by tnjtr2** [comment]
- [x] WO 78516
   -  [x] Wanted to talk to Becky first -- questions on WO
- [x] WO 78554
   -  [x] Wanted to talk to Becky first -- questions on WO
- [x] WO 87185
   - #35 
- [x] WO 102029
   - Waiting on CAD design files from AECOM 
   - #47 
- [x] WO 103587
   -  #31 
   - Already reached out again
- [x] WO 104282
   - Fiber -- Becky took it
- [x] WO 104635
   -  Fiber -- Becky took it
- [x] WO 108424
   - Reached out to AECOM for as-built
- [ ] WO 116015
   -  Delineators -- on hold
- [x] WO 119635
   - Lights under Monona Terrace -- Closed based on feedback from Chad
- [x] WO 120868
   - [x] In Waunakee -- Report to utility? 
- [x] WO 121723
   - Waiting for project to be completed
- [x] WO 122622
   - [x] Reach out again
- [x] WO 124305
   - Waiting for project to be completed 
- [x] WO 124308
   - [x] Does Becky want this one?
- [x] WO 125513
   - Waiting for project to be completed 
- [x] WO 126869
   - [x] Reached out to the shop and engineers about what type of APS they are
- [x] WO 128353
   - [x] Reach out again
- [x] WO 129006
   - [x] Unclear how photocells operate with SL Cabinet timing (Gretchen said she'd reach out)
- [x] WO 132716
   - [x] Unclear how photocells operate with SL Cabinet timing (Gretchen said she'd reach out)
